src/lib/i18n/locales/en/events.ts

```diff
@@ -0,0 +1,31 @@
+export const Namespace = 'events' as const;
+
+export const Strings = {
+  'empty-state-title': 'No Events Match',
+  'empty-state-description':
+    'There are no events that match your filters or selected view. Adjust your filters or view to see your events.',
+  'sort-ascending': 'Sort 1-9',
+  'sort-descending': 'Sort 9-1',
+  'date-and-time': 'Date & Time',
+  'show-elapsed-time': 'Show Elapsed Time',
+  'event-type': 'Event Type',
+  'workflow-events': 'Workflow Events',
+  'category-all': 'All',
+  'category-activity': 'Activity',
+  'category-child-workflow': 'Child Workflow',
+  'category-command': 'Command',
+  'category-local-activity': 'Local Activity',
+  'category-marker': 'Marker',
+  'category-signal': 'Signal',
+  'category-timer': 'Timer',
+  'category-update': 'Update',
+  'category-workflow': 'Workflow',
+  'attribute-group-activity': 'Activity',
+  'attribute-group-parent': 'Parent',
+  'attribute-group-retry-policy': 'Retry Policy',
+  'attribute-group-schedule': 'Schedule',
+  'attribute-group-search-attributes': 'Search Attributes',
+  'attribute-group-summary': 'Summary',
+  'attribute-group-task-queue': 'Task Queue',
+  'attribute-group-workflow': 'Workflow',
```

Comment:
Can we combine `attribute-group-workflow` with `category-workflow` (line 22) since they are both "Workflow"? Or maybe something to add to `common.ts` instead?

Reply:
Yeah I think this is a good use case for the `common` namespace 👍

Reply:
Actually I'm going to leave this as is. In the template file, we call `translate('events', label)`, where `label` maps to these i18n keys. If we use a common key here, we'd have to add some logic to pass in the correct namespace, which seems overkill to reduce a little duplication.